stdlib/3/curses/ascii.pyi

```diff
@@ -0,0 +1,69 @@
+from typing import Sequence, Union, overload
+
+NUL: int
+SOH: int
+STX: int
+ETX: int
+EOT: int
+ENQ: int
+ACK: int
+BEL: int
+BS: int
+TAB: int
+HT: int
+LF: int
+NL: int
+VT: int
+FF: int
+CR: int
+SO: int
+SI: int
+DLE: int
+DC1: int
+DC2: int
+DC3: int
+DC4: int
+NAK: int
+SYN: int
+ETB: int
+CAN: int
+EM: int
+SUB: int
+ESC: int
+FS: int
+GS: int
+RS: int
+US: int
+SP: int
+DEL: int
+
+controlnames: Sequence[int]
```

Comment:
It looks like a `List[str]` to me.

Reply:
Okay, have been a bit fuzzy about when to be more specific and when to be more general ... Then again it could have been because I was just a bit 'fuzzy' in general. Now that you mention it, more specific sounds like the right choice here.